Source/API/CBLJSON.m

```diff
@@ -125,7 +125,11 @@ + (size_t) estimateMemorySize: (id)object {
         // Thanks to DenNukem's answer in http://stackoverflow.com/questions/399527/
         sFormatter = [[NSDateFormatter alloc] init];
         sFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
+#ifdef __IPHONE_8_0
+        sFormatter.calendar = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
```

Comment:
As long as we don't need to maintain the ability to build against previous SDKs.